RELEASE/scripts/autoscend/auto_pre_adv.ash

```diff
@@ -124,6 +124,11 @@ void auto_ghost_prep(location place)
 		}
 		if(canUse(sk)) return;	//we can kill them with a spell
 	}
+	if(auto_haveDarts() && dartEleDmg())
```

Comment:
do we need to check we also have darts available? I'm not very familiar with darts

Reply:
We should always have at least 1 dart because we get a dart after combat.